configure.ac

```diff
@@ -286,8 +286,18 @@ yes_no_try "$enable_legacy_links" "no"
 enable_legacy_links=$?
 AM_CONDITIONAL([BUILD_LEGACY_LINKS], [test $enable_legacy_links -ne $DISABLED])
 
-dnl --with-* options: external software support, and custom locations
+dnl --disable-nls option 
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION(0.18)
```

Comment:
I just discovered there is a better form of the above line, which will allow the latest available files, rather than 0.18 only:

    AM_GNU_GETTEXT_REQUIRE_VERSION([0.18])